productListing/productListing.html

```diff
@@ -0,0 +1,265 @@
+                        amet
+                        consectetur, adipisicing elit. Animi, amet.</p>
+
+                    <div class="price-and-btn">
+                        <strong class="ecommerce-price">₹ 999</strong>
+
+                        <div class="product-btn-section">
+                            <button class="btn btn-primary w-100">Add to Cart</button>
+                        </div>
+                    </div>
+                    
+            </div>
+
+            <div class="product-card">
+                <img src="..\assets\images\products\Marvel\image_38-removebg-preview.webp" alt="" class="ecommerce-image">
+                    
+                    <i class="far fa-heart whishlist"></i>
+                    <h4 class="product-title">FUNKO POP!</h4>
+                    <p class="ecommerce-description">POP TV: Squid Game- Masked Worker Pre-Order Lorem ipsum dolor sit
+                        amet
+                        consectetur, adipisicing elit. Animi, amet.</p>
+                        <div class="price-and-btn">
+                            <strong class="ecommerce-price">₹ 999</strong>
+    
+                            <div class="product-btn-section">
+                                <button class="btn btn-primary w-100">Add to Cart</button>
+                            </div>
+                        </div>
+            </div>
+            
+            <div class="product-card">
+                <img src="..\assets\images\products\Marvel\image_39-removebg-preview.webp" alt="" class="ecommerce-image">
+                    
+                    <i class="far fa-heart whishlist"></i>
+                    <h4 class="product-title">FUNKO POP!</h4>
+                    <p class="ecommerce-description">POP TV: Squid Game- Masked Worker Pre-Order Lorem ipsum dolor sit
+                        amet
+                        consectetur, adipisicing elit. Animi, amet.</p>
+                        <div class="price-and-btn">
+                            <strong class="ecommerce-price">₹ 999</strong>
+    
+                            <div class="product-btn-section">
+                                <button class="btn btn-primary w-100">Add to Cart</button>
+                            </div>
+                        </div>
+            </div>
+
+            <div class="product-card">
+                <img src="..\assets\images\products\Marvel\image_40-removebg-preview.webp" alt="" class="ecommerce-image">
+                    
+                    <i class="far fa-heart whishlist"></i>
+                    <h4 class="product-title">FUNKO POP!</h4>
+                    <p class="ecommerce-description">POP TV: Squid Game- Masked Worker Pre-Order Lorem ipsum dolor sit
+                        amet
+                        consectetur, adipisicing elit. Animi, amet.</p>
+                        <div class="price-and-btn">
+                            <strong class="ecommerce-price">₹ 999</strong>
```

Comment:
I like that you have used strong tag here to make it semantic, I used p tag for that and that is not semantically correct for this usecase. Looks good.